firstvoices/backend/tasks/visibility_tasks.py

```diff
@@ -0,0 +1,71 @@
+from celery import shared_task
+from django.db import transaction
+
+from backend.models import DictionaryEntry, SitePage, Song, Story, StoryPage
+from backend.models.jobs import BulkVisibilityJob, JobStatus
+from backend.models.sites import SiteFeature
+from backend.models.widget import SiteWidget
+from backend.search.tasks.site_content_indexing_tasks import (
+    sync_all_site_content_in_indexes,
+)
+
+
+@shared_task
+def bulk_visibility_change_job(job_instance_id):
+    """
+    Changes the visibility of all dictionary entries in a site.
+    """
+    job = BulkVisibilityJob.objects.get(id=job_instance_id)
+    site = job.site
+    job.status = JobStatus.STARTED
+    job.save()
+
+    # Pause search indexing for the site the job is for
+    if site.sitefeature_set.filter(key="indexing_paused").exists():
+        indexing_paused = site.sitefeature_set.get(key="indexing_paused")
+        indexing_paused.is_enabled = True
+        indexing_paused.save()
+    else:
+        SiteFeature.objects.create(site=site, key="indexing_paused", is_enabled=True)
```

Comment:
This would be a good use for get_or_create. Like:

feature = SiteFeature.objects.get_or_create(site=site, key="indexing_paused")
feature.is_enabled = True
feature.save()